File: src/main/java/io/xol/chunkstories/core/item/armor/ItemArmorLegs.java
//
// This file is a part of the Chunk Stories Core codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package io.xol.chunkstories.core.item.armor;

import java.util.Arrays;
import java.util.Collection;

import io.xol.chunkstories.api.item.ItemDefinition;

public class ItemArmorLegs extends ItemArmor {
    public static final Collection<String> bodyParts = Arrays
            .asList(new String[] { "boneLegRU", "boneLegRD", "boneLegLU", "boneLegLD" });

    public ItemArmorLegs(ItemDefinition type) {
        super(type);
    }

    @Override
    public Collection<String> bodyPartsAffected() {
        return bodyParts;
    }

}

File: src/main/java/xyz/chunkstories/core/item/armor/ItemArmorLegs.kt
//
// This file is a part of the Chunk Stories Core codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.core.item.armor

import java.util.Arrays

import xyz.chunkstories.api.item.ItemDefinition

class ItemArmorLegs(type: ItemDefinition) : ItemArmor(type) {

    override fun bodyPartsAffected(): Collection<String> {
        return bodyParts
    }

    companion object {
        val bodyParts: Collection<String> = Arrays.asList(*arrayOf("boneLegRU", "boneLegRD", "boneLegLU", "boneLegLD"))
    }

}
